libnl-xfrm-3.sym

```diff
@@ -82,6 +82,7 @@ global:
 	xfrmnl_sa_get_encap_tmpl;
 	xfrmnl_sa_get_family;
 	xfrmnl_sa_get_flags;
+	xfrmnl_sa_get_if_id;
```

Comment:
new entries must move to the bottom of the file, to a new `libnl3_12_0` section. See the comment on top of the file.

Reply:
Done. libnl_3_6 to libnl_3_12